package.json

```diff
@@ -128,7 +128,7 @@
     "@guardian/ab-core": "^2.0.0",
     "@guardian/ab-react": "^2.0.1",
     "@guardian/consent-management-platform": "^10.8.0",
-    "@guardian/libs": "^6.0.0",
+    "@guardian/libs": "^6.1.0",
```

Comment:
a link to your changes in guardian/libs in the PR description would be great, just for posterity